src/api/handler.rs

```diff
@@ -0,0 +1,13 @@
+use serde::Serialize;
+use warp::reply::json;
+use warp::{Rejection, Reply};
+
+#[derive(Serialize)]
+struct Health {
+    ok: bool,
+}
+
+// health handler, responds with { ok: true }
+pub async fn health() -> Result<impl Reply, Rejection> {
+    Ok(json(&Health { ok: true }))
```

Comment:
I'm assuming this will eventually expose the actual health of the Vector instance by querying the health of the different active components?

Reply:
Probably, yeah. Right now, this just asserts that the GraphQL handler is alive. There's also a `/health` that returns `{"ok": true}` + a 200 response.